gw_eccentricity/eccDefinition.py

```diff
@@ -310,9 +340,21 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
             get_default_spline_kwargs(),
             "spline_kwargs",
             "utils.get_default_spline_kwargs()")
+        self.rational_fit_kwargs = check_kwargs_and_set_defaults(
+            self.extra_kwargs["rational_fit_kwargs"],
+            get_default_rational_fit_kwargs(),
+            "rational_fit_kwargs",
+            "utils.get_default_rational_fit_kwargs()")
         self.available_averaging_methods \
             = self.get_available_omega_gw_averaging_methods()
         self.debug_level = self.extra_kwargs["debug_level"]
+        # set verbose to debug_level. If verbose is True, then it prints information
+        # of each iteration for rational fits for a given degree.
+        self.rational_fit_kwargs["verbose"] = self.debug_level
+        # keep history of rational fit degree and nonmonotonicity of the corresponding fits
+        self.rational_fit_nonmonotonicity_history = {"pericenters": {}, "apocenters": {}}
+        # Update the degrees used to construct the final fits
```

Comment:
Should the comment be: This is used to keep track and update the degrees used to construct the final fits?

Reply:
Okay, so I am using `rational_fit_nonmonotonicity_history` to keep track of what degrees are being iterated over and the corresponding behaviour, i.e., is it nonmonotonic? And using `rational_fit_degrees` to store the final degree used to build the fits. Instead of using "update the degrees", I changed it to "store degrees" to better convey the purpose.